### DISCUSSION TITLE
No access to Copilot in IntelliJ after successfuly linking device

### DISCUSSION THREAD

**Discussion by Lenieve** [community/community#19046] [Copilot Conversations]
Hey,
I recently gained access and immedietly tried it with VS Code and it worked without issues. Then I tried linking it to intelliJ but it tells me i dont have access. Is there a limit on how many IDEs one can use? 
Thanks in advance.

Im using IntelliJ IDEA Community Edition 2022.1.2 btw. (also, dont know if there is a similar issue, please redirect if that is the case)
Thanks in advance!

**Comment by imageaid** [comment]
Wanted to note that my version of CoPilot has been working in RubyMine (and my other IntelliJ IDEs) for months until today. It "turned off" and I cannot log back in/reactivate even though I have valid access to it and even setup my payment plan for when it goes non-free in August 2022.

**Comment by nat212** [comment]
Having the same issue with WebStorm 2022.1.2 and PyCharm Professional Edition 2022.1.2

**Comment by michalzuk** [comment]
Check your settings at https://github.com/settings/copilot. After selecting preferences and reloading WebStorm, everything started to work again as expected.
